Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -302,6 +302,38 @@ internal List<Result> Search(IEnumerable<Result> results, string searchName)
                 .ToList();
         }
 
+        internal List<Result> InstallFromWeb(string url)
+        {
+            var fileName = url.Split("/").Last();
+            var filePath = Path.Combine(DataLocation.PluginsDirectory, fileName);
+            var plugin = new UserPlugin
+            {
+                ID = "",
+                Name = fileName.Split(".").First(),
```

Comment:
I think it would be better to simply trim the last part (.zip). Also, it is possible that the link doesn't contain file name. Though, I think this won't be a large issue since this feature is mainly for debugging.